config/routes.rb

```diff
@@ -3,6 +3,18 @@
 require 'sidekiq_unique_jobs/web'
 require 'sidekiq-scheduler/web'
 
+class RedirectWithVary < ActionDispatch::Routing::PathRedirect
```

Comment:
Instead of this class, could the redirects in the routes use the block form of the redirect method ... or does that not allow full access to set the vary header? (or look too weird to do it 4x?)

Reply:
I think the block form only allows computing the path, not otherwise change the response